python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -43,6 +44,7 @@ def run_subcommand(argv: list[str]) -> Result[str, str]:
     subparser_record_types = {
         "edit": EditServerConfig,
         "rage": rage.RageConfig,
+        "start": StartServerConfig,
```

Comment:
nit: Instead of a new subcommand, this can be `edit --daemon` or something, loosely following emacs